src/styles/blocks/button.scss

```
@@ -0,0 +1,32 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 270px;
+  width: 100%;
+  height: 60px;
+
+  cursor: pointer;
+  background-color: $secondBackground;
+  color: $colorMain;
+
+  transition-property:color border background-color box-shadow ;
```

your variant doesn’t work. So your transition-duration add transition for all styles

```suggestion
  transition-property: color, border, background-color, box-shadow;
```